### ISSUE TITLE
Improve the content of the CanJS documentation

### ISSUE THREAD

**Issue by chasenlehara** [canjs/canjs#4116]
__TLDR:__ we propose reviewing our current docs to look for:

- pages that don’t have useful content
- broken links
- bad/broken code examples & demos
- other ways we can improve them to make them more useful

## Problem

Our docs have evolved over the years without a thorough review. Inevitably, some docs have deteriorated with broken links, out-of-date examples, and busted demos. Some pages lack good content or examples.

## Solution

Review the most popular pages (maybe top 50?) to:

- Fix broken links, code examples, demos, etc.
- Look for ways to improve the content

Additionally, we should tackle these existing issues:

- [Consistently use const/let in sample code](https://github.com/canjs/canjs/issues/3846)
- [Consistently use ES6 import syntax instead of CJS in sample code](https://github.com/canjs/canjs/issues/3847)
- [Improve the descriptions of (at least) the Core packages](https://github.com/canjs/canjs/issues/4057)
- [Improve docs on loading templates](https://github.com/canjs/can-stache/issues/429)

We should also look at [issues with the “documentation” label](https://github.com/search?o=desc&q=org%3Acanjs+is%3Aopen+is%3Aissue+label%3Adocumentation&s=created&type=Issues).

As a stretch goal, we should consider:

- [Adding a way for users to give us feedback on whether docs are good/useful](https://github.com/canjs/bit-docs-html-canjs/issues/451)
- Making it possible for our demos/examples to open in an online code editor

**Comment by chasenlehara** [comment]
This isn’t something we have to address as part of tackling this proposal, but where we’re already updating code… @code-read brings up a good point that more distinctive variable & property names can make it easier to understand code examples: https://forums.donejs.com/t/generic-property-names-confuse-the-beginner/908

**Comment by justinbmeyer** [comment]
## Priority Guides 

- [ ] Setting Up CanJS 
- [ ] Technology Overview 
- [ ] HTML 
- [ ] Routing 
- [ ] Service Layer 
- [ ] Debugging (Kevin)


## Core 

- [ ] can-component
- [ ] can-define
- [ ] can-stache
- [ ] can-stache-bindings
- [ ] can-route
- [ ] can-value
- [ ] can-route (Kevin)
- [ ] can-route-pushstate (Kevin)
- [ ] can-stache-route-helpers (Kevin)
- [ ] can-fixture 
- [ ] can-query-logic 
- [ ] can-realtime-rest-model 
- [ ] can-rest-model 


## Secondary Guides 

- [ ] Forms (Kevin)
- [ ] Chat 
- [ ] TodoMVC
- [ ] ATM


## Unsorted

- [ ] can-ajax
- [ ] can-assign
- [ ] can-attribute-encoder
- [ ] can-attribute-observable
- [ ] can-bind
- [ ] can-child-nodes
- [ ] can-cid
- [ ] can-compute
- [ ] can-connect
- [ ] can-connect-feathers
- [ ] can-connect-tag
- [ ] can-construct
- [ ] can-construct-super
- [ ] can-control
- [ ] can-data-types
- [ ] can-debug
- [ ] can-define-backup
- [ ] can-define-lazy-value
- [ ] can-define-stream
- [ ] can-define-stream-kefir
- [ ] can-define-validate-validatejs
- [ ] can-deparam
- [ ] can-diff
- [ ] can-dom-data
- [ ] can-dom-data-state
- [ ] can-dom-events
- [ ] can-dom-mutate
- [ ] can-event-dom-enter
- [ ] can-event-dom-radiochange
- [ ] can-event-queue
- [ ] can-fixture
- [ ] can-fixture-socket
- [ ] can-fragment
- [ ] can-globals
- [ ] can-kefir
- [ ] can-key
- [ ] can-key-tree
- [ ] can-list
- [ ] can-local-store
- [ ] can-log
- [ ] can-make-map
- [ ] can-map
- [ ] can-map-define
- [ ] can-memory-store
- [ ] can-namespace
- [ ] can-ndjson-stream
- [ ] can-observation
- [ ] can-observation-recorder
- [ ] can-observe
- [ ] can-param
- [ ] can-parse-uri
- [ ] can-queues
- [ ] can-reflect
- [ ] can-reflect-dependencies
- [ ] can-reflect-promise
- [ ] can-route-hash
- [ ] can-route-mock
- [ ] can-route-pushstate
- [ ] can-set-legacy
- [ ] can-simple-dom
- [ ] can-simple-map
- [ ] can-simple-observable
- [ ] can-stache-converters
- [ ] can-stache-key
- [ ] can-stream
- [ ] can-stream-kefir
- [ ] can-string
- [ ] can-string-to-any
- [ ] can-super-model
- [ ] can-symbol
- [ ] can-validate
- [ ] can-validate-interface
- [ ] can-validate-legacy
- [ ] can-validate-validatejs
- [ ] can-vdom
- [ ] can-view-autorender
- [ ] can-view-callbacks
- [ ] can-view-live
- [ ] can-view-model
- [ ] can-view-nodelist
- [ ] can-view-parser
- [ ] can-view-scope
- [ ] can-view-target
